### ISSUE TITLE
expressions and labels for zone selection

### ISSUE THREAD

**Issue by AdamTheEngineer-dev** [ansys/pyprimemesh#348]
### 🔍 Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### 🐞 Description of the bug

Modifying the elbow example I noticed that the syntax for zone selections varies between different functions.
These should be made to be consistent, preferably using an openly available function.
The zone selection in:
mesh_util.create_zones_from_labels("inlet outlet")
allows space or comma separated values.

mesh_util.volume_mesh(
    volume_fill_type=prime.VolumeFillType.HEXCOREPOLY,
    prism_surface_expression="* !inlet !outlet",
    prism_layers=4,
    growth_rate=1.2
)

a space is allowed to separate search items.

But, in:

display(update=True, scope=prime.ScopeDefinition(model=model, label_expression="inlet, outlet"))
(Borrowed from the Tee-junction example)
the comma **_is_** needed to separate values.

Also, "inlet" selects **_all_** surfaces starting "inlet".
Should this not be "inlet*" ?

This would then follow the globbing rules in the glob standard Python module.

### 📝 Steps to reproduce

Run the standard mixing elbow example with the three lines mentioned changed to include or not include commas.

### 💻 Which operating system are you using?

Windows

### 🐍 Which Python version are you using?

3.7

### 📦 Installed packages

```shell
ansys-api-meshing-prime==0.1.1
ansys-meshing-prime==0.2.0
appdirs==1.4.4
certifi==2022.12.7
charset-normalizer==3.0.1
cycler==0.11.0
fonttools==4.38.0
grpcio==1.51.1
idna==3.4
imageio==2.25.0
kiwisolver==1.4.4
matplotlib==3.5.3
numpy==1.21.6
packaging==23.0
Pillow==9.4.0
pooch==1.6.0
protobuf==3.20.3
pyparsing==3.0.9
python-dateutil==2.8.2
pyvista==0.37.0
requests==2.28.2
scooby==0.7.0
six==1.16.0
typing_extensions==4.4.0
urllib3==1.26.14
vtk==9.2.5
```

**Comment by waltersma** [comment]
APS 774362

**Comment by waltersma** [comment]
limited issue to graphics module, keeping aps label for now

**Comment by waltersma** [comment]
Thank you for the review and feedback!

The comments are addressed in 0.4.0 as follows:

> mesh_util.create_zones_from_labels("inlet outlet")

No longer uses spaces to separate. The following should be used in future: "inlet,outlet"

> mesh_util.volume_mesh(
volume_fill_type=prime.VolumeFillType.HEXCOREPOLY,
prism_surface_expression="* !inlet !outlet",
prism_layers=4,
growth_rate=1.2
)

This is intended as this is a single expression defining "all except inlet and outlet".

> display(update=True, scope=prime.ScopeDefinition(model=model, label_expression="inlet, outlet"))

Intended as both the inlet and outlet labels are required for the display.

> Also, "inlet" selects all surfaces starting "inlet".
Should this not be "inlet*" ?

This has now been fixed and requires "inlet*" to define all surfaces starting with "inlet".
